src/sessionControlClient.js

```diff
@@ -1116,9 +1116,10 @@ class SessionControlClient extends EventEmitter {
             this.midInProcess.doCallback(err);
         }
 
-        this._sendKeepAlive();
         this.inOperation = false;
-        this._sendingProcess();
+
+        // if a serialization error happened, then the streams are destroyed and no longer usable
+        this.close(err);
```

Comment:
I'm a little uncertain about this bit, because I'm not sure why the streams get destroyed... maybe it's something else causing the streams to be destroyed? I just know during testing, I observed them getting destroyed... but I don't think that a serialization error should cause this

Reply:
marking as a todo here https://github.com/RV-Argonaut/node-open-protocol/issues/5